nodejs/week1/avg.js

```diff
@@ -0,0 +1,32 @@
+// process.argv = array whose elements are the parts of the command line. [0] is the runtime, [1] is the script (file) and from [2] onwards, there are arguments passed on the command line
+
+const args = process.argv.slice(2);
+
+const numbers = args.map(string => Number(string));
+check(numbers);
+
+function check (arguments) {
+
+    if (arguments.length === 0) {
+        console.log(`Please provide some numbers!`);
```

Comment:
Here you could use the simple string notation (single quotes). Can you guess why? :)